zipline/gens/brokers/ib_broker.py

```diff
@@ -192,7 +192,10 @@ def subscribe_to_market_data(self,
         self.ticker_id_to_symbol[ticker_id] = symbol
 
         tick_list = "233"  # RTVolume, return tick_type == 48
```

Comment:
This line now only applies to line 198 (reqMktData). Could you please move it to that branch?